lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -3584,4 +3831,30 @@ def air_loop_hvac_has_simple_transfer_air?(air_loop_hvac)
     end
     return simple_transfer_air
   end
+
+  # Determine if the air loop has a return air plenum
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] OpenStudio AirLoopHVAC object
+  # @return [OpenStudio::Model::ThermalZone] OpenStudio thermal zone object of the return air plenum zone
+  #                                          when an air loop uses a return air plenum, nil otherwise
+  def air_loop_hvac_return_air_plenum(air_loop_hvac)
```

Comment:
method name should be air_loop_hvac_return_air_plenum? instead of air_loop_hvac_return_air_plenum

Reply:
Are you suggesting the name of the method should end with at question mark? It is returning a zone object.

Reply:
Maybe the comment needs to change from "Determine if the air loop has a return air plenum" to "Get return air plenum object, if it exists."

Reply:
yep that the description is all that needed to change